src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -565,11 +568,12 @@
         az acr task create -t hello-world:{{.Run.ID}} -n hello-world -r MyRegistry \\
             -c https://github.com/Azure-Samples/acr-build-helloworld-node.git -f Dockerfile \\
             --assign-identity [system] "/subscriptions/<SUBSCRIPTON ID>/resourcegroups/myResourceGroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myUserAssignedIdentitiy"
-  - name: Create a Linux task from a public GitHub repository which builds the hello-world image with a git commit trigger and a timer trigger that runs that task at noon on Mondays through Fridays using the default trigger name.
+  - name: Create a Linux task from a public GitHub repository which builds the hello-world image with a system-assigned managed identity and a timer trigger that runs that task at noon on Mondays through Fridays using the default trigger name.
```

Comment:
If users read the example, I guess they will wonder wonder where the system-assigned managed identity is used for a docker build task. Maybe a multiple-step task is more close to the real user case.

Reply:
@northtyphoon see updated context